apiserver/test/e2e/apiserversdk/cluster_e2e_test.go

```diff
@@ -0,0 +1,46 @@
+package apiserversdk
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/ray-project/kuberay/apiserver/test/e2e"
```

Comment:
Can we avoid importing `apiserver`? The payoff will come when we start deprecating v1. I am OK to add a file to add some test utils which copy some functions from `apiserver/test/e2e`.